bootstrap_cfn/iam.py

```diff
@@ -18,18 +20,299 @@ def __init__(self, aws_profile_name, aws_region_name='eu-west-1'):
+                                                force)
+
+                    upload_success = self.upload_certificate(cert_name,
+                                                             stack_name,
+                                                             ssl_data,
+                                                             force)
+                    if upload_success:
+                        succeeded_count += 1
+                else:
+                    logging.warn("IAM::update_ssl_certificates: "
+                                 "Could not update certificate '%s': "
+                                 "Certificate does not exist remotely" 
+                                 % (cert_name))
+            except AWSQueryConnection.ResponseError as error:
+                logging.warn("IAM::update_ssl_certificates: "
+                             "Could not update certificate '%s': "
+                             "Error %s - %s" % (cert_name,
+                                                error.status,
+                                                error.reason))
+        return succeeded_count
+
+    def check_remote_certificate_exists(self, cert_name, stack_name, ssl_data):
+        """
+        Check to see if the remote certificate exists already in AWS
+
+        Args:
+            cert_name(string): The name of the certificate entry to look up
+            stack_name(string): The name of the stack
+            ssl_data(dictionary): The configuration data for this
+                certificate entry
+
+        Returns:
+            exists(bool): True if remote AWS certificate exists, false otherwise
+        """
+
+        exists = False
+        try:
+            cert_id = "{0}-{1}".format(cert_name, stack_name)
+            logging.info("IAM::check_remote_certificate_exists: "
+                         "Found certificate '%s'.."
+                         % (cert_id))
+
+            # Fetch the remote AWS certificate configuration data
+            # Fetching the response could throw an exception
+            remote_cert_response = self.conn_iam.get_server_certificate(cert_id)
+            remote_cert_result = remote_cert_response['get_server_certificate_response']['get_server_certificate_result']
+            remote_cert_certificate = remote_cert_result["server_certificate"]
+            remote_cert_body = remote_cert_certificate.get("certificate_body",
+                                                           None)
+            remote_cert_chain = remote_cert_certificate.get("certificate_chain",
+                                                            None)
+            remote_cert_key = remote_cert_certificate.get("certificate_key",
+                                                          None)
+            remote_cert_data = \
+            {
+             "cert": remote_cert_body,
+             "chain": remote_cert_chain,
+             "key": remote_cert_key,
+             }
```

Comment:
Can you make this into one block like:
`"cert": remote_cert_certificate.get("certificate_body", None),` etc